### PR TITLE
Adding Ash's Profile

### PR DESCRIPTION
**Pull request by AshAnil**

### CODE REVIEW

File: Ash-Anil/Ash-Anil.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+id: "24b62037-6993-ec11-b400-0022481856af"
+
+qualifications: ""
+
+quote: "Success is the sum of small efforts repeated day in and day out."
+quoteAuthor: "Robert Collier"
+---
+
+[Editing your profile]: https://github.com/SSWConsulting/People/wiki/3.-Editing-your-profile
```
Comment (chrisschultzssw) [inline Ash-Anil/Ash-Anil.md:10]: Remove this line

File: Ash-Anil/Ash-Anil.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Ash Anil is the Network Engineer at SSW. He's a passionate learner gaining experience in Networking infrastructure and adminitration.
```
Comment (chrisschultzssw) [inline Ash-Anil/Ash-Anil.md:12]: Add a comma after "learner"

File: Ash-Anil/Ash-Anil.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+Ash has done his Master's Degree in Network Engineer and currently expereincing a new phase of his life in an IT sector developing skills on Cloud technologies and achieving extensive skills on System Administration.
```
Comment (chrisschultzssw) [inline Ash-Anil/Ash-Anil.md:14]: Change from
> Network Engineer
to
> Network Engineering

Comment (chrisschultzssw) [inline Ash-Anil/Ash-Anil.md:14]: Change from
> in an IT sector
to
> in the IT sector

Comment (chrisschultzssw) [inline Ash-Anil/Ash-Anil.md:14]: Change from
> skills on System Administration
to
> skills in System Administration

**Review by chrisschultzssw — CHANGES_REQUESTED** [review]
Good work! I just added some comments to fix up the grammar.

File: Ash-Anil/Ash-Anil.md (hunk 1 above)
Comment (AshAnil) [reply Ash-Anil/Ash-Anil.md:10]: Thank you Chris 👍

File: Ash-Anil/Ash-Anil.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (AshAnil) [reply Ash-Anil/Ash-Anil.md:14]: hahaha.. I wish GitHub could have got grammarly

**Review by chrisschultzssw — APPROVED** [review]
Noice.

**Pull request merged**